multi-key-example/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "multiple-keys",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.html",
+  "scripts": {
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html"
+  },
+  "dependencies": {
+    "parcel-bundler": "^1.6.1"
+  },
+  "devDependencies": {
+    "@babel/core": "7.2.0",
+    "typescript": "4.4.4"
```

Comment:
how come we have typescript and babel here? I don't think we're using them in any of the code

Reply:
I just copied them from the sandbox example, I can remove it

Reply:
Sweet, just remember that any code you copy and paste is now your code 😉